Doc/library/os.rst

```diff
@@ -222,6 +226,9 @@ process and user.
    :data:`os.environ`, and when one of the :meth:`pop` or :meth:`clear` methods is
    called.
 
+   .. versionchanged:: 3.14
```

Comment:
Maybe put that one below `versionchanged:: 3.9`?

Reply:
done